roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -95,6 +95,20 @@
     - port: 443
       targetPort: "oaproxy"
 
+- name: Set memcached service
```

Comment:
@enoodle I think we need to figure out where this better fits in the logging roles.  @portante has suggested we use memcache for other parts of the logging stack and it would be convenient to set it up correctly the first time.  Maybe it should just live in the 'openshift_logging' role